### PR TITLE
Feature/connect mentions

### PR DESCRIPTION
**Pull request by ArtemLigerko**
WM-Screenshots-20220920181709

### CODE REVIEW

**Review by KirillSerg — APPROVED** [review]
Goog

**Review by DavidKorystov — APPROVED** [review]
lgtm

File: src/containers/MentionsList.tsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+          searchString: `@${user.username}`,
+        })
+      )
+    );
+  };
+
+  return (
+    <>
+      {user._id ? (
+        mentions.docs.length || null ? (
```
Comment (DavidKorystov) [inline src/containers/MentionsList.tsx]: I'm not sure.
```suggestion
        mentions.docs.length ? (
```

File: src/containers/MentionsList.tsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  );
+
+  useEffect(() => {
+    dispatch(
+      tweetsActions.fetchMentions({
+        limit: mentions.limit,
+        nextPage: mentions.nextPage,
+        searchString: `@${user.username}`,
+      })
+    );
```
Comment (VitaliiDenysiuk) [inline src/containers/MentionsList.tsx:29]:
```suggestion
    dispatch(
      tweetsActions.fetchMentions({
        limit: mentions.limit,
        nextPage: mentions.nextPage,
        searchString: `@${user.username}`,
      })
    );
```
you use this block two times. create arrow function and call it.

File: src/containers/MentionsList.tsx [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+  }, []);
+
+  const handleShowMore = () => {
+    return (
+      !mentions.isLoading &&
```
Comment (VitaliiDenysiuk) [inline src/containers/MentionsList.tsx]:
```suggestion
      !mentions.isLoading &&
      mentions.hasNextPage &&
```
if there is not  next page we don't load it

**Review by VitaliiDenysiuk — CHANGES_REQUESTED** [review]

**Review by AndrewKaenko — APPROVED** [review]
Lgtm!

**Review by alexandrtovmach — APPROVED** [review]

**Pull request merged**